pruner.h

```diff
@@ -3,18 +3,24 @@
 
 #include "ir/ir.h"
 
+namespace P4PRUNER {
 
-namespace P4PRUNER{
+#define INFO(x) std::cout << x << std::endl;
```

Comment:
This is a short hand INFO print macro instead of always using std::cout. Unfortunately, the compiler has no support for logging.